include/nbl/video/IGPUDescriptorSet.h

```diff
@@ -45,6 +45,14 @@ class IGPUDescriptorSet : public asset::IDescriptorSet<const IGPUDescriptorSetLa
             uint32_t count;
         };
 
+        struct SDropDescriptorSet
+        {
+            IGPUDescriptorSet* dstSet;
```

Comment:
you already know the `dstSet` you'll drop from by calling `this->dropDescriptors`, its the `this`

Reply:
also call it nullify, cause we not only drop, we nullify as well if we can, they're not supposed to be separate ops

Reply:
hmm we wouldn't need the fast path if https://github.com/Devsh-Graphics-Programming/Nabla/pull/657/files/d608e780494b0a5bde138c9ee679269f3d2b2c59..894a47cd862f678c92991a1ef1861920b4d91f6d#r1512336030 was done

Reply:
Complete suggestion here: https://github.com/Devsh-Graphics-Programming/Nabla/pull/662